packages/action-group/test/action-group.test.ts

```diff
@@ -934,6 +968,36 @@ describe('ActionGroup', () => {
         expect(secondElement.selected, 'second child selected').to.be.false;
     });
 
+    it('accepts role attribute override', async () => {
```

Comment:
Based on your description here, could you clarify a use case where a consuming application wouldn't be making a mistake in doing this?

Reply:
A toolbar that contains one or more `sp-action-group` elements with different selection modes might use `role="group"`, or `role="presentation"` on the descendant groups.

```html
      <div role="toolbar" aria-label="Text Formatting" style="height: 32px; display: flex; gap: 6px">
        <sp-action-group aria-label="Text Style" selects="multiple" role="group" compact emphasized>
          <sp-action-button label="Bold" value="bold">
            <sp-icon-text-bold slot="icon"></sp-icon-text-bold>
          </sp-action-button>
          <sp-action-button label="Italic" value="italic">
            <sp-icon-text-italic slot="icon"></sp-icon-text-italic>
          </sp-action-button>
          <sp-action-button label="Underline" value="underline">
            <sp-icon-text-underline slot="icon"></sp-icon-text-underline>
          </sp-action-button>
        </sp-action-group>
        <sp-divider
              size="s"
              style="align-self: stretch; height: auto;"
              vertical
          ></sp-divider>
        <sp-action-group aria-label="Text Align" selects="single" compact emphasized>
          <sp-action-button label="Left" value="left" selected>
            <sp-icon-text-align-left slot="icon"></sp-icon-text-align-left>
          </sp-action-button>
          <sp-action-button label="Center" value="center">
            <sp-icon-text-align-center slot="icon"></sp-icon-text-align-center>
          </sp-action-button>
          <sp-action-button label="Right" value="right">
            <sp-icon-text-align-right slot="icon"></sp-icon-text-align-right>
          </sp-action-button>
          <sp-action-button label="Justify" value="justify">
            <sp-icon-text-align-justify slot="icon"></sp-icon-text-align-justify>
          </sp-action-button>
        </sp-action-group>
        <sp-divider
              size="s"
              style="align-self: stretch; height: auto;"
              vertical
          ></sp-divider>
        <sp-action-group aria-label="List Style" selects="multiple" role="group" compact emphasized>
          <sp-action-button label="Bulleted" value="bulleted">
            <sp-icon-text-bulleted slot="icon"></sp-icon-text-bulleted>
          </sp-action-button>
          <sp-action-button label="Numbering" value="numbering">
            <sp-icon-text-numbered slot="icon"></sp-icon-text-numbered>
          </sp-action-button>
        </sp-action-group>
        <sp-divider
              size="s"
              style="align-self: stretch; height: auto;"
              vertical
          ></sp-divider>
        <sp-action-group role="presentation" compact>
          <sp-action-button disabled label="Copy" value="copy">
            <sp-icon-copy slot="icon"></sp-icon-copy>
          </sp-action-button>
          <sp-action-button disabled label="Paste" value="paste">
            <sp-icon-paste slot="icon"></sp-icon-paste>
          </sp-action-button>
          <sp-action-button disabled label="Cut" value="cut">
            <sp-icon-cut slot="icon"></sp-icon-cut>
          </sp-action-button>
        </sp-action-group>
      </div>
```